scripts/publish-if-needed/index.js

```diff
@@ -33,7 +33,7 @@ function getHighestVersion(version1, version2) {
 // Exit the script with a success message
 function exitPublish() {
   info("Publishing");
-  exec(`(cd ${packagePath} && yarn publish)`);
+  exec(`(cd ${packagePath} && npm publish)`);
```

Comment:
For anyone wondering why, `yarn publish` runs interactively and expects user input. We're reverting for now to the deploy process we're accustomed to.